packages/amplify_datastore/example/ios/unit_tests/GraphQLRequestExtensionTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
*nit:* for consistency with other headers in the repo

```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```